### DISCUSSION TITLE
read datum of cross-section

### DISCUSSION THREAD

**Discussion by nnbach2022** [DHI/mikeio1d#32] [Q&A]
Hi,
Is it possible to get datum value out of each cross-section in xns11 file?
Thanks.

**Comment by JesperGr** [comment]
The coordinate system (WKT-Projection string) that an xns11 file belongs to is not stored in the xns11 file but in the associated .mhydro file or MIKE+ database.

A cross section file can store coordinates, but need not to. Cross sections created from the MIKE 11 user interface most often do not store coordinates. If a cross section does not store coordinates, the coordinates must be inferred from the location (ID of reach/link/branch + chainage) in the network that it is connected to.
